### PR TITLE
feat: Add support for TensorboardTimeSeries and writing scalar data points.

### PR DESCRIPTION
**Pull request by sasha-gitg**

### CODE REVIEW

File: google/cloud/aiplatform/tensorboard/tensorboard_resource.py [hunk 1]
```diff
@@ -782,3 +785,331 @@ def list(
...
+            parent_resource_name_fields={
+                Tensorboard._resource_noun: tensorboard_id,
+                TensorboardExperiment._resource_noun: tensorboard_experiment_id,
+            },
+            project=project,
+            location=location,
+        )
+
+        gapic_tensorboard_time_series = gca_tensorboard_time_series.TensorboardTimeSeries(
+            display_name=display_name, description=description, value_type=value_type,
```
Comment (yfang1) [inline google/cloud/aiplatform/tensorboard/tensorboard_resource.py:1022]: Could you pass in plugin_name as well like https://github.com/googleapis/python-aiplatform/blob/d72a254e97cf74f3fdd55a32a4af86737243593a/google/cloud/aiplatform/tensorboard/uploader.py#L438 ?

The scalar plugin name: https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/scalar/metadata.py#L21

Comment (yfang1) [reply google/cloud/aiplatform/tensorboard/tensorboard_resource.py:1022]: ... and if you plan to write non-scalar data you might need plugin_content as well, which I don't have much knowledge on how to generate.

Comment (sasha-gitg) [reply google/cloud/aiplatform/tensorboard/tensorboard_resource.py]: Added

**Review by yfang1 — APPROVED** [review]

**Review by morgandu — APPROVED** [review]

File: google/cloud/aiplatform/tensorboard/tensorboard_resource.py [hunk 2]
```diff
@@ -774,6 +780,362 @@ def list(
...
+            time_series_data (Dict[str, float]):
+                Required. Dictionary of where keys are TensorboardTimeSeries display name and values are the scalar value..
+            step (int):
+                Required. Step index of this data point within the run.
+            wall_time (timestamp_pb2.Timestamp):
+                Optional. Wall clock timestamp when this data point is
+                generated by the end user.
+
+                If not provided, this will be generated based on the value from time.time()
+        """
```
[the remark this answers is not in the document]
Comment (sasha-gitg) [reply google/cloud/aiplatform/tensorboard/tensorboard_resource.py:818]: Updated.

**Pull request closed without merging**